cg/services/orders/utils.py

```diff
@@ -1,8 +1,16 @@
-from cg.server.dto.orders.orders_response import Order as PydanticOrder
+from cg.server.dto.orders.orders_response import Order as PydanticOrder, OrdersResponse
 from cg.store.models import Order as DatabaseOrder
 
 
-def parse_order(order: DatabaseOrder) -> PydanticOrder:
+def _parse_order(order: DatabaseOrder) -> PydanticOrder:
     return PydanticOrder(
         customer_id=order.customer_id, ticket_id=order.ticket_id, order_date=order.order_date
     )
+
+
+def create_orders_response(orders: list[DatabaseOrder]) -> OrdersResponse:
+    return OrdersResponse(orders=[_parse_order(order) for order in orders])
+
+
+def create_order_response(order: DatabaseOrder) -> PydanticOrder:
```

Comment:
I changed so I did not use "PydanticOrder" so maybe we should refrain from that here as well?